generator/transit_generator.cpp

```diff
@@ -43,12 +45,17 @@ string GetFileName(string const & filePath)
 
 /// \brief Reads from |root| (json) and serializes an array to |serializer|.
 template <class Item>
-void SerializeObject(my::Json const & root, string const & key, Serializer<FileWriter> & serializer)
+void SerializeObject(my::Json const & root, string const & key, Serializer<FileWriter> & serializer,
+                     function<void(vector<Item> &)> handler = nullptr)
```

Comment:
Please (very briefly) document the handler.

Reply:
Done